script-library/jupyterlab-debian.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
+# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/jupyterlab.md
+# Maintainer: The VS Code and Codespaces Teams
+#
+# Syntax: ./jupyter-debian.sh
+
+set -e
+
+JUPYTER_LAB_VERSION=${1:-"latest"}
```

Comment:
I think it would make more sense to just call this `VERSION`, as it lines up with the option name in the `devcontainer-features.json` file above.

And then somewhere around this part of the file: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/container-features/src/feature-scripts.env#L14

Add in something like:

```
_VSC_INSTALL_JUPYTERLAB="-jupyterlab-debian.sh ${_BUILD_ARG_JUPYTERLAB_VERSION:-latest}"
```

Again, that `_BUILD_ARG...` is auto-generated by the user setting that version property in their `devcontainer.json`, and code-gen'd here: https://github.com/microsoft/vscode-remote-containers/blob/main/src/node/containerFeatures.ts#L180-L193